packages/api/src/domain/medical/address.ts

```diff
@@ -7,8 +9,28 @@ export type Address = {
   state: USState;
   zip: string;
   country?: string;
+  coordinates?: Coordinates;
```

Comment:
We should not have have our domain depending on other layers (in this case, external). Better to have `Coordinates` (or `GeoCoordinates`) on our domain.